presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -356,6 +356,8 @@ Property Name                                          Description
 ``iceberg.io.manifest.cache.max-content-length``       Maximum length of a manifest file to be considered for          ``8388608``
                                                        caching in bytes. Manifest files with a length exceeding
                                                        this size will not be cached.
+
+``iceberg.split-manager-threads``                      Number of threads to use for generating iceberg splits.         ``Number of available processors``
```

Comment:
One more thing, should configuration properties add here? See `Configuration Properties` chapter.

Reply:
Great question, and good catch! I have a question for you:

Is the ``iceberg.split-manager-threads`` property related to [Manifest File Caching](http://prestodb.io/docs/current/connector/iceberg.html#manifest-file-caching)? 
* If yes, then I think it should stay where it is. 
* If no, then I suggest moving the ``iceberg.split-manager-threads`` property to the table in [Configuration Properties](http://prestodb.io/docs/current/connector/iceberg.html#configuration-properties) that you noticed. 

Thanks!

Reply:
It's not related to `Manifest File Caching`, so I think it's better to be placed in `Configuration Properties`.

Reply:
@hantangwangd @steveburnett I read the documentation and found that I had indeed added the wrong place, and I have corrected it. Thanks.